db/access.php

```diff
@@ -51,6 +51,16 @@
         ]
     ],
 
+    'mod/applaunch:manage' => [
```

Comment:
this doesn't seem to be used anywhere?

Reply:
Removed. I thought we needed it for the _{action)_instance callbacks.